src/ggml-cuda.cu

```diff
@@ -4782,12 +4784,13 @@ static void rope_f32_cuda(const float * x, float * dst, const int ncols, const i
     rope_f32<<<block_nums, block_dims, 0, stream>>>(x, dst, ncols, p0, p_delta, p_delta_rows, theta_scale);
 }
 
-static void rope_glm_f32_cuda(const float * x, float * dst, const int ncols, const int nrows, const float p, const float block_p, const float theta_scale, cudaStream_t stream) {
+static void rope_glm_f32_cuda(const float * x, float * dst, const int ncols, const int nrows, const float p0,
+                              const float p_delta, const int p_delta_rows, const float theta_scale, const int n_ctx, cudaStream_t stream) {
     GGML_ASSERT(nrows % 4 == 0);
```

Comment:
```suggestion
    GGML_ASSERT(ncols % 4 == 0);
```
I implemented the original check incorrectly and it seems to have been copied for GLM RoPE.

Reply:
Good catch! Fixed.